validator/sawtooth_validator/journal/publisher.py

```diff
@@ -35,6 +37,30 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class TransactionSender(object):
```

Comment:
From the consensus point of view it is a TransactionSender, a consensus algorithm hands it a set of transactions and they get sent.